apps/deploy-web/src/utils/sdl/sdlGenerator.ts

```diff
@@ -3,6 +3,25 @@ import yaml from "js-yaml";
 import { ExposeType, ProfileGpuModelType, ServiceType } from "@src/types";
 import { defaultHttpOptions } from "./data";
 
+export const buildCommand = (command: string) => {
+  if (command.trim() === "sh -c") {
+    return ["sh", "-c"];
+  }
+
+  const lines = command.split("\n");
+
+  if (lines.length > 1 || command.startsWith("sh -c")) {
```

Comment:
I might be missing the point here but there might be other executables out there then `sh`, should this be addressed?

Reply:
I haven't tested this in a long time

Reply:
ok, do you think it's safe to merge without considering this?